protected/humhub/modules/user/behaviors/Followable.php

```diff
@@ -186,20 +208,33 @@ public function getFollowers($query = null, $withNotification = false, $returnQu
         return $query->all();
     }
 
+    /**
+     * Get a query of objects which the owner object follows
+     *
+     * @param ActiveQueryUser $query e.g. for limit the result
+     * @return ActiveQueryUser
+     */
+    public function getFollowingQuery($query = null)
```

Comment:
I think the only reason we have to keep the `$query` parameter here is because of the old deprecated functions. I don't like the idea of adding a new function with a parameter we do not really need/want. Maybe we can remove the $query parameter and find a better solution. We should maybe just ignore the `$query` param in the deprecated functions and refactor all the usages. I assume the $query param is hardly used anyways.

Reply:
Commit https://github.com/humhub/humhub/pull/4879/commits/5394e00c89fce92a6220d916d5a555a5f127fb8e